vaadin-radio-button-flow-demo/src/main/java/com/vaadin/flow/component/radiobutton/demo/RadioButtonGroupView.java

```diff
@@ -68,6 +71,27 @@ private void basicDemo() {
         addCard("Basic usage", radioGroup);
     }
 
+    private void helperDemo() {
+        // begin-source-example
+        // source-example-heading: Helper text
+        RadioButtonGroup<String> radioGroup = new RadioButtonGroup<>();
+        radioGroup.setLabel("Label");
+        radioGroup.setItems("Option one", "Option two", "Option three");
+        radioGroup.setHelperText("Helper text");
+
+        RadioButtonGroup<String> radioGroupWithHelperSlotted = new RadioButtonGroup<>();
```

Comment:
Slotted is more of an implementation detail, I'd rather make it about the demoed feature `HelperComponent`

Reply:
Renamed it.